content/docs/state-and-lifecycle.md

```diff
@@ -302,84 +302,84 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](http://codepen.io/gaearon/pen/amqdNA?editors=0010)
+[**Przetestuj kod na CodePen**](http://codepen.io/gaearon/pen/amqdNA?editors=0010)
 
-Now the clock ticks every second.
+Teraz timer powinien już tykać co sekundę.
 
-Let's quickly recap what's going on and the order in which the methods are called:
+Podsumujmy, co dzieje się w powyższym kodzie i w jakiej kolejności wywoływane są metody:
 
-1) When `<Clock />` is passed to `ReactDOM.render()`, React calls the constructor of the `Clock` component. Since `Clock` needs to display the current time, it initializes `this.state` with an object including the current time. We will later update this state.
+1) Kiedy element `<Clock />` przekazywany jest do funkcji `ReactDOM.render()`, React wywołuje konstruktor komponentu `Clock`. Jako że `Clock` będzie wyświetlać aktualny czas, musi on zainicjalizować `this.state` obiektem zawierającym aktualną datę. Później ten stan będzie aktualizowany.
 
-2) React then calls the `Clock` component's `render()` method. This is how React learns what should be displayed on the screen. React then updates the DOM to match the `Clock`'s render output.
+2) Następnie React wywołuje na komponencie `Clock` metodę `render()`. W ten sposób uzyskuje informację, co powinno zostać wyświetlone na stronie. Gdy otrzyma odpowiedź, odpowiednio aktualizuje drzewo DOM.
```

Comment:
Metoda `render` nie jest wywoływana "w" komponencie?

Reply:
Wywoływana jest z zewnątrz. W JS w podejściu obiektowym metody są zwykłymi funkcjami, które mają dowiązane `this`. Wywołanie można by zapisać jako `render.call(Clock)`, więc stawiam na "wywołuje na komponencie" lub "wywołuje dla komponentu".

Reply:
Spoko, sposób w jaki jest napisane po angielsku (`component's method` - metoda komponentu, a następnie `Clock's render` - render "clocka") sugeruje, że jest ona jednak `w` komponencie, ale nie będę się kłócił.

Reply:
Nie no, można to zdanie napisać po prostu jako "Następnie React wywołuje metodę `render()` komponentu `Clock`." i po problemie :-)